.github/workflows/release.yml

```diff
@@ -3,6 +3,23 @@ name: Build and Tests
 on: [push]
```

Comment:
`release` should only run on `branches: ['main']`. The bump to `0.1.1` on this branch just published to `npm` when it should only run after the PR's merged